modules/services/blueman-applet.nix

```diff
@@ -12,7 +12,7 @@ with lib;
         since it requires running 'blueman-mechanism' service activated via dbus.
         You can add it to the dbus packages in system configuration:
```

Comment:
Does this paragraph also need to be updated?

Reply:
up